docs/basics/101-136-cheatsheet.rst

```diff
@@ -0,0 +1,187 @@
+by a lower-case letter in front of the command name.
+
+Dataset operations
+""""""""""""""""""
+
+   +--------------+---------------+----------------------------------------------------------+
+   | Command      |General flags  |    Description                                           |
+   +==============+===============+==========================================================+
+   |              |  [h/f/d/D]    | create a new dataset from scratch.                       |
+   |  **create**  |               | If executed within a dataset and                         |
+   |              |               | the ``-d/--dataset`` flag, it is                         |
+   |              |               | created as a subdataset.                                 |
+   +--------------+---------------+----------------------------------------------------------+
+   |    ``datalad create [-c <config-procedure>] PATH``                                      |
+   +--------------+---------------+----------------------------------------------------------+
+   |              |  [h/d/D/r/R]  | install an existing dataset from a path/url/open data    |
+   |  **install** |               | collection. When using ``-g/--get-data``, all dataset    |
+   |              |               | content is obtained with a ``get`` operation. Providing  |
+   |              |               | ``-d`` installs a dataset as a subdataset.               |
+   +--------------+---------------+----------------------------------------------------------+
+   |``datalad install -s/--source URL/PATH [-g/--get-data] [PATH]``                          |
+   +--------------+---------------+----------------------------------------------------------+
+   |              |  [h/d/D/r/R]  | Get any dataset content (files/directories/subdatasets)  |
+   |  **get**     |               | Will get *directory* content recursively by default, but |
+   |              |               | not *subdataset* content. Specify the label of a data    |
+   |              |               | source (e.g. sibling) with ``-s/--source``.              |
+   +--------------+---------------+----------------------------------------------------------+
+   |``datalad get [-s/--source <label>] PATH``                                               |
+   +--------------+---------------+----------------------------------------------------------+
+   |              |  [h/d/r/R/m]  | Save the current state of a dataset. Use ``-u/--updated``|
+   |   **save**   |               | to leave untracked files untouched, and ``--to-git`` to  |
+   |              |               | save modifications to Git instead of Git-annex.          |
+   |              |               |                                                          |
+   +--------------+---------------+----------------------------------------------------------+
+   |``datalad save [-u/--updated] [PATH]``                                                   |
```

Comment:
That is where examples with explanation could be handy to describe cases where PATH points to submodule